src/main.rs

```diff
@@ -170,3 +172,25 @@ fn main() {
         Err(err) => error(err.description()),
     }
 }
+
+fn transform_args_with_exec() -> Vec<OsString> {
+    let target = {
+        let mut ex = OsString::new();
+        ex.push("-exec");
+        ex
+    };
+
+    if !std::env::args_os().any(|v| v == target) {
```

Comment:
If we are going to traverse the arguments anyway, I think we can leave out this check. Just try to replace `-exec` with `--exec` below.

Reply:
I've removed this, but please see my comment below